jetty-maven-plugin/src/it/jetty-start-gwt-it/beer-client/src/main/java/org/olamy/App.java

```diff
@@ -44,9 +44,9 @@
      * The message displayed to the user when the server cannot be reached or
      * returns an error.
      */
-    private static final String SERVER_ERROR = "An error occurred while "
-        + "attempting to contact the server. Please check your network "
-        + "connection and try again.";
+    private static final String SERVER_ERROR = "An error occurred while " +
+        "attempting to contact the server. Please check your network " +
+        "connection and try again.";
```

Comment:
Again, odd that this has nothing to do with shutdown. Perhaps you are missing a merge from jetty-10.0.x?

Reply:
I started getting checkstyle failures, so I had to fix this.  I think it was a problem with the merge of the change to apply checkstyle to tests?